sos/plugins/openstack_aodh.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2017 Red Hat, Inc., Sachin Patil <psachin@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import os
+from sos.plugins import Plugin, RedHatPlugin
+
+
+class OpenStackAodh(Plugin, RedHatPlugin):
+    """OpenStack Alarm service"""
+    plugin_name = "openstack_aodh"
+    profiles = ('openstack', 'openstack_controller')
+
+    packages = (
+        'openstack-aodh-api',
+        'openstack-aodh-listener',
+        'openstack-aodh-notifier',
+        'openstack-aodh-evaluator,'
+        'openstack-aodh-common')
```

Comment:
Should be in sos list style (which you use pretty consistently throughout the rest of the plugin...). I'll fix during merge.